### DISCUSSION TITLE
Set Buttons to switch colors for annotations

### DISCUSSION THREAD

**Discussion by RennexTech** [sumatrapdfreader/sumatrapdf#3899] [Ideas]
I wish you could add the ability for me to set my keyboard extra keys like numkeys to be used for different annotation colors. Instead of going here to change colors:

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/139177099/1961d38e-3247-4583-9765-91e20864acbe)

I could just press the keys that I have set for that specific color and it works.

**Comment by RennexTech** [comment]
I found the discussion, sorry for repetition.

**Reply by animanoir** [reply]
where's the discussion? this is **absolutely essential**.

**Reply by GitHubRulesOK** [reply]
@animanoir what discussion the manual has all the instruction for changing numkeys for shortcuts descriptions and adding text icons to toolbar. thus you can on toolbar have for example <kbd>[R]</kbd><kbd>[G]</kbd><kbd>[B]</kbd> or any other text and those be assigned to say <kbd>[1]</kbd><kbd>[2]</kbd><kbd>[3]</kbd> NOTE they dont have to be on toolbar just set the keys you want to the colour you want.
![image](https://github.com/user-attachments/assets/6bb69dc9-797d-422f-877c-d1b353e0031e)
